apps/ledger-live-desktop/src/renderer/components/WebPlatformPlayer/TopBar.jsx

```diff
@@ -146,10 +149,16 @@ const WebPlatformTopBar = ({
     const webview = webviewRef.current;
 
     if (webview) {
+      if (shouldDisplayBackToLedgerLiveApp) {
+        const url = webview.getURL();
+        const isLedgerLiveAppUrl = url.includes(manifest.url);
```

Comment:
I would rather compare the domains for equality, no ?
It would be more robust to changes we may do like : 
- adding a query parameter to manifest.url : ...buy.app?sandboxes=true
- manifest.url not being the root url : ...buy.app/intro

Reply:
Done!